README.md

```diff
@@ -1,72 +1,106 @@
+  - 2.4.3 &nbsp; Launch WLS Ubuntu and create a <strong>  username (`airflow`) & password (`airflow`) </strong> when prompted
 
-  - 2.3.7 &nbsp; Open a comamnd line in Windows (CMD) and excute the following command to make sure that Ubuntu has been set as a default WSL:
+  - 2.4.4 &nbsp; In the WSL terminal window go to `/home/airflow` and clone the repo:
 
-       `wsl --setdefault Ubuntu`
+        cd /home/airflow && git clone <'Airflow 2 repository'>
 
-  - 2.3.8 &nbsp; Install (if not already installed ) and launch <strong> Visual Studio Code </strong>
+  - 2.4.5 &nbsp; On Windows 10, install the latest available version of `Docker Desktop`: https://docs.docker.com/get-docker/
 
-  - 2.3.9 &nbsp; From the VS code extension tab, search and install a new plugin <strong> `Remote WLS` </strong>
+  - 2.4.6 &nbsp; Once installed, launch `Docker Desktop`, go to <strong> Settings --> Resources --> WSL INTEGRATION and toggle "Ubuntu". Once done, click the "Apply & Restart" button  </strong>
 
-  - 2.3.10 &nbsp; On  <strong> Visual Studio Code </strong>, you now see a green WSL indicator in the bottom left corner, click on it and choose <strong> Open Folder in WSL </strong>. Windows will prompt you to select a folder, provide the follwing path to a folder: <strong> ` \\wsl$\ubuntu\home\airflow ` </strong> , and choose the folder with the Airflow code: (<strong> airflo2-local-cicd </strong>)
+  - 2.4.7 &nbsp; Open a comamnd line in Windows (CMD) and excute the following command to make sure that Ubuntu has been set as a default WSL:
 
-  - 2.3.11 &nbsp; Open a terminal session in VS code `(Menu Terminal -- New Terminal)` and run the WLS docker installation script:
+        wsl --setdefault Ubuntu
 
-       `chmod +x ./helpers/scripts/docker-wls.sh && sudo ./helpers/scripts/docker-wls.sh`
+  - 2.4.8 &nbsp; Install (if not already installed ) and launch <strong> Visual Studio Code </strong>
 
-  - 2.3.12 &nbsp; Proceed with the installation and initialization steps (<strong> section #3 and #4 </strong>)
+  - 2.4.9 &nbsp; From the VS code extension tab, search and install a new plugin <strong> `Remote WLS` </strong>
+
+  - 2.4.10 &nbsp; On  <strong> Visual Studio Code </strong>, you now see a green WSL indicator in the bottom left corner, click on it and choose <strong> Open Folder in WSL </strong>. Windows will prompt you to select a folder, provide the follwing path to a folder: <strong> ` \\wsl$\ubuntu\home\airflow ` </strong> , and choose the folder with the Airflow code: (<strong> airflo2-local-cicd </strong>)
+
+  - 2.4.11 &nbsp; Open a terminal session in VS code `(Menu Terminal -- New Terminal)` and run the WLS docker installation script:
+
+        chmod +x ./helpers/scripts/docker-wls.sh && sudo ./helpers/scripts/docker-wls.sh
+
+  - 2.4.12 &nbsp; Proceed with the installation and initialization steps (<strong> section #3 and #4 </strong>)
 
 <br/>
 
```

Comment:
Reaplce "Dependencies to solve before you go" with "Customizing Airflow Settings"